test/integration.test.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+const { test, teardown } = require('tap')
+const uuid = require('uuid')
+const Fastify = require('fastify')
+const nodeVault = require('node-vault')
+
+const FastifySecretsHashiCorp = require('../lib/fastify-secrets-hashicorp')
```

Comment:
you can require the default entry point of the plugin

```suggestion
const FastifySecretsHashiCorp = require('..')
```